xmlserializer/XmlDocSource.cpp

```diff
@@ -133,6 +129,13 @@ _xmlDoc* CXmlDocSource::getDoc() const
     return _pDoc;
 }
 
+bool CXmlDocSource::isParsable() const
+{
+    // Check that the doc has been created
+    // Implicit pointer->bool conversion
```

Comment:
well please remove this useless comment, I'm not sure this is the right place to learn C++